integration_tests/test_gravity.py

```diff
@@ -343,20 +343,23 @@ def test_multiple_attestation_processing(gravity):
             gravity.contract, erc20, recipient, amount, KEYS.values()
         )
 
+        current_height = cli.block_height()
         def check_gravity_balance():
             """
             check the all attestation are processed at once by comparing
             with previous block balance
             """
             nonlocal previous
-            current = cli.balance(eth_to_bech32(recipient), denom=denom)
+            nonlocal current_height
+            current = cli.balance(eth_to_bech32(recipient), denom=denom, height=current_height)
```

Comment:
what would be the result if height is larger than latest one, we should check against with current height?

Reply:
we are incrementing the height by one at the end of the loop, so we are checking each block from first current_height

Reply:
to be more safe, I will get the height before sending the tx